tests/unit/modeling.py

```diff
@@ -216,9 +213,7 @@ def __init__(self,
         self.out_features = out_features
         self.fused_gelu = False
         self.fused_tanh = False
-        if isinstance(act,
-                      str) or (sys.version_info[0] == 2 and isinstance(act,
-                                                                       unicode)):
+        if isinstance(act, str):
```

Comment:
Why was the or statement removed here?

Reply:
`unicode` is undefined when running on Python 3 (it's Python 2-only), and this project uses f-strings in `setup.py`, so it probably can't even build wheels for anything below Python 3.6. This check is therefore useless and removing it is likely better than suppressing the flake8 error.